src/Bicep.Core/Registry/IArtifactRegistry.cs

```diff
@@ -89,7 +90,7 @@ public interface IArtifactRegistry
         /// Returns description for the module.
         /// </summary>
         /// <param name="reference">The module reference</param>
-        Task<string?> TryGetDescription(ArtifactReference reference);
+        Task<string?> TryGetDescription(ModuleSymbol module, ArtifactReference reference);
```

Comment:
nit: Parameter 'module' has no matching param tag in the XML comment for 'IArtifactRegistry.TryGetDescription(ModuleSymbol, ArtifactReference)' (but other parameters do) CS1573

Reply:
Also about this interface since we are refactoring maybe its worth separating it into two separate interfaces, one that is module specific and one that isn't to have better separation of concerns.

Reply:
I addressed this by removing the original comment - as I didn't feel it was adding value. I personally don't like the idea of filling out comments "for completeness" - I prefer to only add them if they are truly needed.